### ISSUE TITLE
Windows

### ISSUE THREAD

**Issue by jasonrdavis** [dcarosone/scrape-cassini#3]
Hi, 

I have the Perl environment installed on Windows, but I can't find the required modules:

libanyevent-http-perl 
libdatetime-format-natural-perl
liburi-perl

I tried looking for them in CPAN, and installing them in the Perl shell using this command:

C:\Windows\System32>cpanm Module::libanyevent-http-perl
! Finding Module::libanyevent-http-perl on cpanmetadb failed.
! Finding Module::libanyevent-http-perl () on mirror http://www.cpan.org failed.
! Couldn't find module or a distribution Module::libanyevent-http-perl

Any suggestions?

Thanks!

Jason Davis
The Planetary Society

**Comment by jasonrdavis** [comment]
I figured this out. The sequence, using CPANM, that worked was:

cpanm AnyEvent --force
cpanm DateTime::Format::Natural
cpanm URI
cpanm AnyEvent::HTTP

**Comment by dcarosone** [comment]
Sorry about that, Jason.   Those names from the README example are the Ubuntu packages that wrap the perl modules, because every packaging system is different, alas.  Glad you worked it out.

I'll update the readme to be a bit clearer.

Meanwhile, I've just pushed a fix for the corrupted image download on windows (issue #4).

**Comment by dcarosone** [comment]
Jason, can you comment on why you needed the `--force` option for `AnyEvent`?  Ordinarily it would have been pulled in as a dependency from `AnyEvent::HTTP`, which is why I don't list it explicitly for the Ubuntu example.

I presume this failed for some reason, and so you manually added it with `--force` to work around that. Did it give any specific reason why it failed?

**Comment by jasonrdavis** [comment]
Ah, I assumed that AnyEvent:HTTP was a dependency of AnyEvent -- not the other way around. So originally I was only installing AnyEvent and the script wasn't working. I got an install error once and then forced it, and then installed AnyEvent:HTTP. This was the procedure I used for my machine and I advised Emily to do the same. 

So, it's likely you only need to install AnyEvent:HTTP and it will work. Sorry, I'm a little rusty on this type of work.
